.github/workflows/ci.yml

```
@@ -11,10 +11,10 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
-      - uses: cachix/install-nix-action@v17
+      - name: Install Nix
+        uses: determinatesystems/nix-installer-action@main
```

```suggestion
        uses: DeterminateSystems/nix-installer-action@main
```

(and everywhere else)